install.rdf

```diff
@@ -1,40 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-     xmlns:em="http://www.mozilla.org/2004/em-rdf#">
-
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
   <Description about="urn:mozilla:install-manifest">
-    <em:id>exchangecalendar@extensions.1st-setup.nl</em:id>
-    <em:version>4.0.0-beta4</em:version>
+    <em:id>exchangecalendar@ExchangeCalendar</em:id>
+ <em:version>4.0.0-beta5</em:version>
```

Comment:
Now, at build time, the `<em:version>` line is directly filled with the content of the new `VERSION` file (that's work of the `sed` part of the new Makefile). Do you think about something else ?

For the `<em:id>` part, I am not sure that's a good idea to update it.
Indeed, I think that Thunderbird won't erase older addons (the one from 1st-setup and Ericsson) if we modify it.